src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
```

Comment:
you can simplify without using template string
```suggestion
        My name is {name}
```